clinica/pipelines/t1_linear/t1_linear_pipeline.py

```diff
@@ -0,0 +1,263 @@
+"""T1 Linear - Clinica Pipeline.
```

Comment:
Missing # coding: utf-8 (this is corrected in PR #63 if you generate new pipeline.

BTW, take the opportunity to delete the comments which were automatically generated by clinica (they were designed to help the developper. If you didn't feel that way, it could be nice to have your feedback ^^).